backend/fms_core/migrations/0018_v3_1_0.py

```diff
@@ -184,5 +198,9 @@ class Migration(migrations.Migration):
             name='created_at',
             field=models.DateTimeField(auto_now_add=True, help_text='Date the instance was created.'),
         ),
+        migrations.RunPython(
```

Comment:
This should not be moved there ... auto_now_add fields can't be changed once they are created. the idea is to initialize them while they are just fields with a default value, then use alterfield to activate auto_now_add  for new records.